CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ This changelog keeps track of work items that have been completed and are ready
 ### Fixes
 
 - **General**: Increase ScaledObject polling interval to 15 seconds ([#799](https://github.com/kedacore/http-add-on/issues/799))
+- **General**: Add new user agent probe ([#862](https://github.com/kedacore/http-add-on/issues/862))
```

Comment:
```suggestion
- **General**: Add new user agent probe ([#862](https://github.com/kedacore/http-add-on/issues/862))
- **General**: Increase ScaledObject polling interval to 15 seconds ([#799](https://github.com/kedacore/http-add-on/issues/799))
```

Reply:
@maxmoeschinger , please commit the suggestion 🙏 Static checks are failing due to it